app/views/complaints/_complaints_open_new_linked_item.html.erb

```diff
@@ -0,0 +1,13 @@
+<div class="display-flex flex-align-center">
```

Comment:
[question] This is the kind of approach that I could take if we want me to keep combining partials. Since this was such a small chunk it seemed like a good test case, and it's still fairly readable.  Not sure that some of the bigger partials are going to be nearly as readable?  But I am happy to give it a try if y'all want.

Reply:
I think it could make sense to do that for the _issue_tta_report and _issue_monitoring_review at least, but maybe that can/should also be punted until editing of the reviews is enabled

Reply:
There is definitely a lot in terms of editing / deleting that will need to be in the shared partials, so punting makes sense to me!